docs/management_api.rst

```diff
@@ -26,26 +26,74 @@ Pinot Management Console
 ------------------------
 
 There is a REST API which allows management of tables, tenants, segments and schemas. It can be accessed by going to
-``http://[controller host]/help`` which offers a web UI to do these tasks, as well as document the REST API.
+``http://[controller host]/help`` which offers a web UI to do these tasks, as well as document the REST API. The below
+is the screenshot of the console.
 
-For example, list all the schema within Pinot cluster:
+  .. figure:: img/pinot-console.png
+
+For example, to list all the schemas within Pinot cluster:
 
   .. figure:: img/list-schemas.png
 
-Upload a pinot segment:
+To rebalance segments of a table across servers:
 
-  .. figure:: img/upload-segment.png
+  .. figure:: img/rebalance-table.png
 
 
-Pinot-admin.sh
+pinot-admin.sh
 --------------
 
-It can be used instead of the ``pinot-admin.sh`` commands to automate the creation of tables and tenants.
+It can be used instead of the ``pinot-admin.sh`` commands to automate the creation of tables and tenants. The script
```

Comment:
What can be used instead of pinot-admin.sh? (The section title seems to be pinot-admin.sh...)

Reply:
Addressed in https://github.com/apache/incubator-pinot/pull/3991.